tox.ini

```diff
@@ -1,5 +1,6 @@
 [tox]
-envlist = py27, py35, py36, py37, py38, py39, py310, pypy, pypy3
```

Comment:
If we are removing py35 and py36 from the testing matrix we should reflect that in the README (we list tested versions under `Installation`)